Contribute/index.md

```diff
@@ -0,0 +1,140 @@
+
+```cmd
+git remote add upstream https://github.com/umbraco/UmbracoDocs/
+git fetch upstream
+git rebase upstream/master
+```
+
+**Step 3: Creating a PR**
+
+If you do this locally and then push it to your fork you will have a synced up fork to start working with! Once you have made some changes and you are happy with the result, you can create a pull request (you may have to rebase again and resolve merged conflicts if a lot of things has been merged in since your last sync).
+![Creating a PR](images/pull-request.png)
+
+And that is all you need to do to create a fork, sync it and make a pull request to the main repository! 
+
+
+#### Labels for PRs
+- `Dependant On Release` - this label is for documentation pull requests that are made in advance of a new Umbraco release, will be merged in once the changes have been released
+- `need-OUR-update` - this label is used for PR's awaiting some new functionality being added to the Our Umbraco website
+- `D-Team` - this label is usually used when the PR is about something with Umbraco Core that the documentation team is not familiar with, meaning it requires a review from the Umbraco development team
+- `Umbraco Cloud`, `Umbraco Forms`, etc - if the issue is specific to something other than Umbraco CMS it will get the relevant Umbraco product label
+
+## Markdown conventions
+The Umbraco Documentation uses Markdown for all of the documentation - but more precisely we use the CommonMark specification, read more about the difference [here](https://commonmark.org/).
+
+### Structure
+For the documentation project, each individual topic is contained in its own folder.
+Each folder must have an `index.md` file which links to the individual sub-pages, if images are used, these must be in `images` folders next to the .md file referencing them relatively.
+
+* topic
+	* images
+		* images.jpg
+	* Subtopic
+		* images
+		* index.md
+	* index.md
+	* otherpage.md
+
+### Images
+Images are stored and linked relatively to .md pages, and should by convention always be in an `images` folder. So to add an image to `/documentation/reference/partials/renderviewpage.md` you link it like so:
+
+	![My Image Alt Text](images/img.jpg)
+
+And store the image as `/documentation/reference/partials/images/img.jpg`
+
+Images can have a maximum width of **800px**. Please always try to use the most efficient compression, `gif`, `png` or `jpg`. No `bmp`, `tiff` or `swf` (Flash).
+
+### External links
+Include either the complete URL, or link using Markdown:
+
+	https://yahoo.com/something
+
+	or
+
+	[yahoo something](https://yahoo.com/something)
+
+
+### Internal links
+If you need to link between pages, always link relatively, and include the .md extension.
+
+	[Umbraco.Helpers](Umbraco.Helpers.md)
+
+	or
+
+	[Umbraco.Helpers](../../Reference/Umbraco.Helpers.md)
+
+### Formatting code
+Indent your sample with a single tab, which will cause it to be rendered as `<pre><code>` tags.
```

Comment:
4 spaces, no tabs

(as stated in our editorconfig https://github.com/umbraco/UmbracoDocs/blob/master/.editorconfig#L12)